### ISSUE TITLE
Merging Memphis and West Memphis

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#985]
In GitLab by @kdjstudios on Jul 19, 2018, 14:25

**Submitted by:** "Miya Harris" <miya.harris@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-07-19-52714

**Server:** Internal

**Client/Site:** Memphis and West Memphis

**Account:** ALL

**Issue:** 

Several months ago, we merged all the triad billing into one monthly bill. Doing this, we ran into some issues with data loss and had to manually reenter all accounts.

 

We will be merging Memphis and West Memphis (SABilling account) next week wanted to your input on the best way to handle this task without losing any data or running into any errors.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jul 19, 2018, 14:27

I wrote:

> Thank you for reaching out. The SA Billing support team has already begun discussing the merge since the two VCC servers have already been combined and has already caused issues with billing.  I have just sent out an email with a draft plan of action and proposed a call to discuss the merge process for SAB to insure we can complete the SA Billing updates in a timely manor. Please feel free to follow up with that email. We will keep this ticket open until the updates are complete. 

Email I sent out separately:

> Over the last few month we have had multiple SA Billing issues and concerns arise from the VCC Merge between Memphis and West Memphis. We would like to set a plan to insure that we minimize and hopefully eliminate any further issues due to this merge. Below I have outlined a draft for a basic plan of action. We would also like to schedule a call to allow for discussion and improvement of the plan. I will suggest Monday the 23th at 2pm EDT to have this call.
> 
> 1.	Discussion Call mentioned above to discuss plan and insure all teams are correctly informed.

> - a.	We know Memphis and West Memphis are on one server in VCC, now.

>  - i.	How are the VCC Portal reports setup to merge/split the accounts accordingly? Example “Billing Export Report”

> - b.	Confirm all accounts in Motion’s Contact List, Motion’s Voice Route, and Open Answer are setup with the site number prefixed. Example 1208888 or 0448888

> - c.	Operations will run the exceptions report to confirm all accounts are setup correctly. They will need to make all corrections until this report is blank.

>  - i.	This must be completed before proceeding to make any adjustments in SA Billing and before the next billing cycle.

> 2.	Confirm if accounts are also to be merged into one site in SAB. – This will be discussed during the call mentioned above. 

> - a.	If yes; then we need to address how to transfer/link to historical data after the move of accounts in SAB.

>  - i.	This will be handled by the SAB support team. 

> - b.	If no, then we simply need to update all of the resource IDs accordingly. (Preferred method)

>  - i.	This will be handled by the SAB support team. 

> 3.	Test and confirm exceptions in SAB on the Merge (Development) server before releasing the update to resource IDs to production; to prevent any unforeseen errors. 

> - a.	Release the updated account numbers to production.

> 4.	Finish!
> 
> Please let us know if you have any questions or concerns.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jul 30, 2018, 15:44

@smasih @pchaudhary @tim.traylor 

We were able to discuss this internally and have provided the attached plan of action. [Memphis_Correction_Plan_7-30-18.docx](/uploads/f5d5d6cad49c1e3fe96f5e882f34404d/Memphis_Correction_Plan_7-30-18.docx)

Please note to summarize the first item needed to be completed by the Aavaz Team(Step # 3 on the document). Please let me know if you have any questions or concerns. 

- The SAB support team will update all account numbers and account resource IDs in SAB with the site number prefixed accordingly. 
 - a.	Accounts will be Moved from the West Memphis site to the Memphis site in SAB. Each account should be moved to the correct billing cycle accordingly. Master->Master; NCSM->NCSM; etc. 
 - b.	Since both sites are VCC, all accounts should be using the same billing codes and activities.
 - c.	Will update account contact info as to prevent any testing items sent to actual clients. 
 - d.	If we have any issues or concerns during this we will update operations accordingly.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jul 31, 2018, 02:36

@kdjstudios @tim.traylor 

Questions on Point # 3

> - The SAB support team will update all account numbers and account resource IDs in SAB with the site number prefixed accordingly.

* Do we need to update **all** accounts ..? means not just Wes Memphis and Memphis all sites ..?
* Updating the resource ID will affect usage cycle calculation because on usage report there is not site number prefixed on resource_id column.
 * if so we need to update VCC script to ignore site number
* There are accounts in Memphis with **044** and **120** as account number prefix. What do we need to do about those accounts..?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jul 31, 2018, 08:50

@smasih 

Thank you for reaching out with your concerns.

- Update only West Memphis and Memphis site accounts.
- The resource ID is being updated on the usage report, thus why we are needing to update the accounts. 
- If the account already has a prefix, please ignore it but keep a spreadsheet list of these accounts so we and operations can double check them.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Jul 31, 2018, 09:41

@smasih Please be sure that the previous usage is brought over and is updated with the new resource ID so we don't lose their usage history.  Thx

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jul 31, 2018, 09:45

Please be sure to update the resource ID and account numbers with the prefix before moving them over to the Memphis sites.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 3, 2018, 08:28

@smasih had wrote in the other ticket: 

> fixed the 54 accounts on production, but still waiting for an approval on prefixing site number on resource ID and account number

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 14, 2018, 09:28

I will follow up with operations to see if they are ready to continue with the testing on the Merge server.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 14, 2018, 12:31

I wrote:

> Just a quick update:
> 
> - The support team was able to fix the changes made by operations to the production server. We are awaiting an confirmation from Haydee in ticket  2018-07-31-2379.
> 
> - The support team is still waiting for the mapping document from operations that was requested in the previous email. Once we have received that, the support team will continue to make the updates on the DEVELOPMENT server; we will then have operations check and confirm those updates are correct beofre releasing to the PRODUCTION server.  
> 
> Please let us know if you have any questions or concerns.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 14, 2018, 15:25

Cori wrote:

> Hi Kyle- Please transfer all of the fees over. Thanks Cori

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 17, 2018, 08:09

@smasih @pchaudhary @tim.traylor 

Please complete the Merge of West Memphis accounts and fees into the Memphis site. We are needing to have this completed, tested, and released before their next billing cycle.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 17, 2018, 08:31

@kdjstudios 

We need the mapping because if we just simply transfer the fees over (which are linked to West Memphis site fees) system will start behaving weirdly.

Alternatively, we can only transfer accounts and fess has to be assigned manually.

once mapping is received this task will take 1-2 hr (including our testing)

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 17, 2018, 08:31

changed time estimate to 2h

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 17, 2018, 08:35

@smasih @pchaudhary @tim.traylor 

Why can you not just transfer the fees over? What will behave weirdly? Please explain and Discuss with @tim in your morning meetings. (Post update here too)

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 18, 2018, 02:04

@kdjstudios 

Account Activities are linked to Activities on Site in other words site activities are Parent to account activities.

If we move an account from site to another all the account activities are also carried along with it.

Now, their link to their parent is broken and it will stay broken until unless we assign them a new parent (Memphis site activity).

As a result, system will throw an error while billing and on account page.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Aug 20, 2018, 06:29

@kdjstudios Hi Kyle - Sumeet is going to post the list of fees for West Memphis that do not have a match in memphis.  Pls give that list to Ops so they can map the fees for us & Aavaz can know which of the new fees to use on the Memphis side.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 20, 2018, 08:14

Juan Wrote:

> This is just to confirm we are still on track to merge sites two weeks from now and if there is anything we should do on our end.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 20, 2018, 08:23

I just reached out to @tim.traylor to explain why we can not just move all the fees/activities over; so I can explain this to Cori. He also does not see why we can not just move them all over and is reaching out to Aavaz for further clarification.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 20, 2018, 08:59

@kdjstudios @tim.traylor 

here is the file with recurring/additional fee mapping.

west stands for `West Memphis, AR` in the file and memphis is for [+Memphis, TN+]

[memphis_mapping3_without_id.csv](/uploads/05f71d016cc09512ec2bc86f789d276b/memphis_mapping3_without_id.csv)

What we can do is... create a new additional/recurring fee on site level for those which are not mapped.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 20, 2018, 09:21

@kdjstudios 

Account Recurring Fees are linked to Additional Fees on Site in other words site additional fess are Parent to account recurring fees.

If we move an account from site to another all the account recurring fees are also carried along with it.

In our case we will move West Memphis accounts to Memphis and West Memphis accounts are linked to Additional Fees on Site West Memphis

Now, their link to their parent is broken and it will stay broken.

If somehow we assign a new Parent(Memphis additional fee) to the moved West Memphis accounts

If the link is kept broken, system will keep throwing errors while billing and on account page.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 20, 2018, 10:53

Keeping this in Development. This is a time sensitive ticket and must be completed before their next billing cycle, Which is in two weeks according to Juan http://gitlab.aavaz.biz/AnswerNet/SABilling/issues/986#note_25405.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 21, 2018, 08:15

@kdjstudios 

I have one more question.

West Memphis offices: `"AR", "NCSM", "VOICEMAIL", "OTHER"`  
Memphis TN: `"ARST", "M", "NCSM", "VOICEMAIL", "OTHER"`  

What about Offices ..? should we just create a new office for transferred accounts or map them ??

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 21, 2018, 08:19

@smasih 

I would think; for now we should add the site prefix to each. So "120 - NCSM" and "044 - NCSM" would the the offices. This way operations can at a time they see fit either reduce these down into one or they can maintain the separation. 

@tim.traylor do you agree?

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Aug 21, 2018, 09:17

@smasih Please prefix '120 - ' for the offices being moved over from West Memphis.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Aug 21, 2018, 09:22

@smasih THis is to be on merge only right now.  Ops will then review and give us the green light to perform the final update in production.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 21, 2018, 09:23

@smasih @pchaudhary 

Per discussion: we will be not using a mapping document but will be coping all existing fees over from West Memphis to Memphis. This will be completed tomorrow on **MERGE**. We will then inform operations to check and confirm everything. Once approved by operations we can then release to Production.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 22, 2018, 08:28

@kdjstudios @tim.traylor 

merged west memphis and memphis TN 

changes are on staging.sabilling.com (merge and mege2 were busy)

I created new recurring/additional fees, **taxes**, **late charges** (with 120 prefix) on those entries which were not mapped.

once again [here is the list of mapped additional fees of west memphis with memphis TN ](http://gitlab.aavaz.biz/AnswerNet/SABilling/uploads/05f71d016cc09512ec2bc86f789d276b/memphis_mapping3_without_id.csv) I created new fees on Memphis TN which were not mapped

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 22, 2018, 09:37

@smasih @pchaudhary @tim.traylor 

I was just about to send out to have operations start testing, but I spot checked an account and it does not look like the email addresses have been updated. Please go through the staging server and insure it is ready for testing. We do not want to send any info to actual clients!!

![image](/uploads/41acb39b400c568ca3ea8063fa395a47/image.png)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 22, 2018, 10:12

I also just ran the account details report and it does not look like any of Memphis account's resource IDs were updated. 

[account_details____3_.pdf](/uploads/a4eeb34065d97fa1b63b6fc350cdcffd/account_details____3_.pdf)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 22, 2018, 10:43

Received update in Whatsapp that this is ready for testing now.

I wrote:

> Thank you for reaching out. The support team initially had ran into a concern with the ability to merge all the fees over from West Memphis to Memphis. They have now successfully been able to complete the transfer for accounts, sub billing cycles, recurring fees, offices, taxes and late charges successfully. These changes have been made on our development server for operations to test and confirm all is working as requested. Please have all of operations visit https://staging.sabilling.com/ and login with your same SAB login credentials and review and test the merged info. We have listed a few of the most common processes and items we recommend to test.
> 
> NOTE: The support team has replaced all client contact info so nothing should be sent out to clients when you send emails, faxes, etc. on the https://staging.sabilling.com/ development server. **Please do not perform any tests on the production server. 
> 
> 1. Run 2-3 billing cycle processes. Please use the usage upload, revert, change dates, etc. to insure all functionality is working. 
> 2. Spot check at least 5-10 accounts. Please confirm all their fees, activities, etc are setup and functioning properly. Be sure to check their billing history, ledger, etc are all functioning as intended. Also be sure to check the account's resource IDs are correct.
> 3. Run 3-5 reports. Please insure that the reports are working correctly.
> 4. Update accounts, customers, etc. Please confirm you are able to change account and customer info without issue. NOTE: Please revert your changes back to the correct data as this is the information the support team will use to update the production server.
> 5. Anything else you use on a regular basis please test and confirm is working.  
> 
> In order to help keep this ticket and email thread uncluttered. If you find any issues or concerns; please, simply open a new ticket by emailing support@sabilling.com using the subject line "SAB - Staging Memphis Merge - (ISSUE)". Where (ISSUE) is a short description of the issue. In the email body please detail out further what steps you performed and what issue or concern occurred. If possible, please include screenshots too. Please feel free to include anyone else on these emails too. 
> 
> Please let us know if you have an questions or concerns.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 22, 2018, 10:49

I also wrote:

> I forgot to add in order to maintain completion before the next billing cycle; I would believe everyone would greatly appreciate if this initial testing could be completed over the next 48 hours (By Friday). This way we still have roughly 5 days for the support team to work on an issues or concerns before the next Billing cycle, which is the 9/1/18 I believe. Please let us know if there are any questions or concerns.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 30, 2018, 08:14

@smasih @pchaudhary @tim.traylor 

Juan wrote:

> Testing billing for this, I am getting over 1,100 missing accounts. I believe it's caused due to the resource id's having a hyphen after the site number such as "120-18494." We will have to either remove the hyphen from SAB or add the hyphen to the billing export.  Please advise, thank you. 


Please look into this ASAP as this is marked as Important.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Aug 31, 2018, 02:28

I am little confused here as per the initial requirement we had to prefix resourceID with 044 and 120 (depending on their origin)

so, do we need to remove 120 from the resource id 

**or**

 we need to remove 044 as well from the resource id of west memphis accounts in Memphis TN site.

this should quick but first, we should be clear on what to be done.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Aug 31, 2018, 06:10

@smasih Hi Sumeet - there are asking for the hyphen to be removed, not the prefixing number.  So change "120-" to "120" and "044-" to "044".

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Aug 31, 2018, 07:02

Hi [Tim] ( @tim.traylor ),  

We have removed the hyphen from the resource IDs of accounts.  

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 31, 2018, 08:19

@smasih @pchaudhary @tim.traylor 

So if I am understanding correctly, the Aavaz team did not follow the directions laid out in http://gitlab.aavaz.biz/AnswerNet/SABilling/issues/986#note_23562 and also added a hyphen to the site prefix for the resource IDs which caused this?

I wrote:

> May you please retest. The support team was able to find a discrepancy and they believe it is now resolved. Thank you and we look forward to your response.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 31, 2018, 12:20

Jesus wrote:

> Missing accounts have reduced to 532, the resource codes on the billing export don't seem to be matching with SAB.
> 
> Please find export attached along with the list of missing accounts currently showing.

[Test+Missing+Accounts.xlsx](/uploads/50c1132eaaf730cd8c9a3234dca4ed57/Test+Missing+Accounts.xlsx)

[Billing+Export+_2868_29.csv](/uploads/88f77a4c69ad70ab4ed3426b0574a791/Billing+Export+_2868_29.csv)

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Sep 1, 2018, 05:40

There is still a mismatch b/w existing resourceIDs and those in usage file.

[memphis_missing4.csv](/uploads/c9c1efb6c0332bbe273be8866760e67d/memphis_missing4.csv)

This file contains usage resourceID **vs** existing resourceID in SAB.

Please have a look.

we can filter those accounts which don't have "120" prefix but not sure about others.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 4, 2018, 08:11

@smasih @pchaudhary @tim.traylor 

So I see two issues here.

1. There are still accounts in SAB that we did not update to included the site prefix?
2. There are as expected accounts part of the usage file:
 - That do not have a site prefix (This needs to be updated on operations side)
 - That are not in SA Billing at all (This needs to be updated on operations side, after the support team confirms that the account is not in SAB at all)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 4, 2018, 09:56

The Aavaz team is going to check and update the list accordingly, as it would appear it was ran on the wrong server.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 4, 2018, 14:20

Juan wrote:

> Billing needs to be completed this week. Is there a timetable on when the production server will be ready for billing to be completed. Memphis and West Memphis still show as separate sites on SAB.

I wrote:

> Thank you for reaching out. The support team is looking into the issue with the amount of missing accounts you reported in your testing on 8/29/18; they were able to reduce the number of missing accounts by about 50% by Aug 31st. The support team will continue to provide updates as further progress is made. 
> 
> Please keep in mind that we reached out for operations to begin testing on 8/22/18; which we did not receive any feedback for over a week (8/29/19); which has been the main cause of delay in progress on this topic. Thanks and we will continue to keep you updated.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 4, 2018, 14:20

@smasih @pchaudhary @tim.traylor 

Updating this to Urgent, as their billing is needing to be completed by the end of the week.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Sep 5, 2018, 05:48

Hi [Kyle] ( @kdjstudios ),   

Sorry for the inconvenience caused to you but the file uploaded by [Sumeet] (@smasih ) on the above [comment] (http://gitlab.aavaz.biz/AnswerNet/SABilling/issues/986#note_26198) was tested on Merge & not accurate with the data too. Here is the file with the data tested on the Staging server with the proper data. 

 [memphis_missing.csv](/uploads/10aae30211e5a9eb4268ea1da402eedb/memphis_missing4.csv)

We dug into the issue & below are our findings: 

*  Many accounts do have **double entries on the Usage file**. One includes the site number as prefix & one has not due to which the entry without the site number as prefix comes under the missing accounts report.  
  Eg: Resource ID `118943` -  (Dr. Purnima Sarkar MD) had mentioned twice on the usage file & only once it includes site number as a prefix.

*  Many accounts **do not** includes the site number as a prefix on the Usage file due to which system throws those accounts under the missing accounts report.  
  Eg: Resource ID `997` - (Dr. Sudesh Bananji MD PLLC) does not include the site number as a prefix on the Usage file which should be `120997`.

**Note** - SAB system recognize the **resource_id** column of the usage file to check the resource ID of the account in SAB.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 5, 2018, 08:15

@tim.traylor 

So it would appear that the VCC Billing Export report is separating out the accounts; is there any way we can update the report to merge them together for just this month? Or will we need to have someone program a script to do that?

@pchaudhary @smasih 

Once we are able to consolidate the report correctly, then operations should be able to rerun their upload step and hopefully at that point it will only have the accounts that are truly missing accounts.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 5, 2018, 08:26

I wrote:

> The support team was able to further determine that there are two main concerns currently happening. We will describe each below.
> 
> 1. The first concern is that the VCC Billing Export report appears to be using old account numbers/resource IDs, this was not expected to happen. Currently we have tech services looking into a way to resolve this.
> 2. The second concern is that there are still a rather larger list of accounts that are not in SA Billing; about 170 accounts. We have attached the list for operations to review and insure all of those accounts are true exceptions. If not, please update the account to insure SA Billing finds a matching record. 
> 
> We will continue to keep you updated on the VCC Billing Export report progress. Please keep us updated on fixing the exception accounts.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 6, 2018, 08:09

Juan wrote:

> Please find list attached of the accounts I have worked on. Everything in yellow is showing in the missing accounts due to them being duplicates and having a different resource code. I have notated the resource code that is on SAB on every duplicate so far. The rest have been corrected to match the export. 
> 
> I will continue to work on them and send and update but it seems that the remaining accounts will have the same issues.

[Copy+of+Memphis+Merged+Missing+Accounts.xlsx](/uploads/2ea6b6ada99eda1054151d67d1909d59/Copy+of+Memphis+Merged+Missing+Accounts.xlsx)

@smasih @pchaudhary @tim.traylor 

Judging from the start of that list, it would appear that a majority these accounts did not get updated in SAB? Please look into this as well. As it will require updates from the Aavaz Team.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 6, 2018, 08:12

I wrote:

> The support team is currently working on updating the VCC Report and once we are also able to confirm where all of those accounts are we should be ready to have operations test again. 

Just to keep everyone in the loop. We have Nadine looking into updating the report data which I am hoping we will have done by EOD. So all that needs to be completed will be insuring all accounts in SAB for West Memphis and Memphis had their account # and Resource ID updated.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 6, 2018, 15:35

Haydee wrote:

> SAB Support Team- we needed to have finalized billing as of Monday.  We see the merge on the testing part, when are we merging the live one?
> 
> I found some issues with the file we uploaded I corrected what needed to be corrected, not the only thing that come out on the exception file are West Memphis accounts that have not been merged in the live SAB.
> 
> We need to get this billing completed and finalized ASAP.

Miya:

> I also went into both OA and Motion and made the changes that were needed on the 170 accounts. I also noticed that some of the accounts you listed (from Memphis) are correct, the system removed the 0 in front of "044"
> 
> Can you please move forward with testing so that we can complete billing?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 7, 2018, 08:36

I wrote to Haydee:

> Thank you for reaching out. As pointed out in a previous email we have lost a week of testing due to operations not testing for a full week. (8/21/18 to 8/29/18). The support team is continuing to work and update as quickly as humanly possible. 
> 
> The support team has found the attached list of account which operations failed to update the "bill to" account numbers in Open Answer. Please have operations update those accounts. Thanks and we look forward to your response.
> 
> In regards to the VCC Portal Billing Export Report; Nadine is still working on updating the data so the report will export the merged data correctly. We will continue to keep you updated. 

I wrote to Miya:

> Thank you for updating those accounts, we greatly appreciate it. As stated in a previous email we currently have Nadine working on updating the data so the Billing Export report will export the merged data correctly. We are also having the support team continue to double check the updates operations has made and will continue to post updates on any discrepancies they find accordingly. I will also notify the support team to insure if the "0" in front of the resource IDs is required and if so, we will inform Nadine to make those updates accordingly. Thank you for bringing that to our attention.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 7, 2018, 12:39

So just a recap and update for everyone.

- The Aavaz team has completed all updated in SAB and we do not at this point see any issues in regards to data in SAB. :+1: 


- We have found issues with historical data and the VCC Billing Report that are causing accounts to not be found in SAB
 - We have issued an HD ticket for Joe A to double check that databases in Motion and OA to insure operations has not missed any updates. This should insure there are no accounts with their old Resource ID. (Note operations should be able to use the exceptions report, but it does not appear to be working. I submitted a separate HD ticket to address that)
 - HD: http://www.servicedesk.answernet.com/profiles/ticket/2018-09-06-31535/conversation


 - We have issued a HD ticket for Nadine to update the historical records for the accounts that had their resource IDs updated. This should result in the VCC Billing Export Report to only have one record per account's resource ID.
 - HD: http://www.servicedesk.answernet.com/profiles/ticket/2018-09-05-97451/conversation



1. The VCC Billing Report export is still showing duplicate entries for the same resource ID. (I don't know if these has any true effect on the import but would presume it will only use one of the records) 
 - This should be addressed with Nadine's HD ticket.

2. The VCC Report is not including the "0" in the "044" prefix. (Only about 20 of the accounts are showing on the missing accounts page in SAB, so I would presume SAB is able to match them with out the "0", and that these 20 are actually not in SAB at all) 
 - This should not be an issue. as it appears SAB is not showing all of the "044" site's accounts as missing. But is only showing a small number, which would lead us to believe it is just these accounts are not actually in SAB.
 
3. I presume there are still entries in the historical database that are showing the old resource ID; because they are still in the VCC Report. (There is about 100 accounts that still show without a prefix on their resource ID 
 - This should be resolved with first Joe confirming there are not accounts with old Resource ID's and then also with Nadine confirming all historical data is updated. 

4. Still waiting on Joe to run the databases checks to insure if the issue in step 3, is a historical data issue that is causing issues with the VCC Report or if operations failed to update those accounts.  
 - This should be address with Joe's HD ticket.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Sep 10, 2018, 01:05

@kdjstudios 



> The VCC Report is not including the "0" in the "044" prefix. (Only about 20 of the accounts are showing on the missing accounts page in SAB, so I would presume SAB is able to match them with out the "0", and that these 20 are actually not in SAB at all)

yes, posting scripts can handle "0" prefix issues.

**Is there anything we can do to help.?**

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Sep 10, 2018, 09:12

@kdjstudios @tim.traylor 

migration of West Memphis accounts to Memphis TN was successful (as per my testing)

however, I took the backup.

here is the list of transferred accounts [memphis_merge1.csv](/uploads/9a1ef3863e69a4f3eaffd0f2b48bb517/memphis_merge1.csv)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 10, 2018, 13:18

@smasih 

Thanks, I dont believe there is nothing left at them moment for the Aavaz Team. I will keep updated posted.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 11, 2018, 08:53

After communications from Tim, Nadine, Haydee, Cori and I Billing was finalized. I would suggest we keep this ticket open for 48 hours in case there are any concerns; due to this being a large undertaking and the amount of previous concerns. 

Haydee wrote:

> Billing has been finalized.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 13, 2018, 11:43

Marking for close

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Sep 13, 2018, 11:51

closed
